packages/datagateway-search/src/searchBoxContainer.component.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
```

Comment:
I note that `searchBoxContainer` has been changed but `searchBoxContainerSide` hasn't. I think `containerSide` will also need to be provided the same changes